README.md

```diff
@@ -75,9 +75,12 @@ NOTE:
 | CephFS | Dynamically provision, de-provision File mode RWO volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=14.2.2)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWX volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
-|        | Creating and deleting snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
+|        | Creating and deleting snapshot                            | Alpha          | >= v3.1.0          |  >= v1.0.0       | Octopus (>=v15.2.3)  | >= v1.17
```

Comment:
don't we have functionality in 15.0.0?

Reply:
snap info is after v15.2.2+ is what I am seeing.